File: help/eng_gb/exercicios_da_ficha_4/Ficha04_005/Ficha04_005.cs
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha04_005 {

    //Main Function
    static void Main() {
        Console.Write("Number \t: " );
        int num = int.Parse(Console.ReadLine());
        if(num == 0 ){
            Console.Write("Null" );
        }else{
            if(num > 0 ){
                Console.Write("Positive" );
            }else{
                Console.Write("Negative" );
            }
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:27

File: help/eng_gb/exercicios_da_ficha_4/Ficha04_005/Ficha04_005.cpp
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha04_005 {

    public:
        int main() {
            cout << "Number \t: ";
            int num;
            cin >> num;
            if(num == 0 ){
                cout << "Null";
            }else{
                if(num > 0 ){
                    cout << "Positive";
                }else{
                    cout << "Negative";
                }
            }
            return 0;
        }
};// end of class Ficha04_005
// Execute main function
int main(){
    Ficha04_005 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:27
